packages/dds/merge-tree/src/zamboni.ts

```diff
@@ -44,6 +44,16 @@ export function zamboniSegments(
 			segmentToScour?.segment?.parent &&
 			segmentToScour.segment.parent.needsScour !== false
 		) {
+			const refs = segmentToScour.segment.localRefs ?? [];
```

Comment:
i'm not sure zamboni is the right place to do this, as it won't work once we have sidedness, where do we update movedSeqs?

Reply:
I actually haven't been able to find where movedSeqs is updated, but I would guess that the reference cleanup needs to happen near the calls to zamboni in insertSegments and obliterateRange.

Reply:
https://github.com/microsoft/FluidFramework/blob/fff9bab5a5e3e0c76e3e8edbf87301b912260795/packages/dds/merge-tree/src/mergeTree.ts#L1050

Reply:
https://github.com/microsoft/FluidFramework/blob/fff9bab5a5e3e0c76e3e8edbf87301b912260795/packages/dds/merge-tree/src/mergeTree.ts#L1219

Reply:
Ah, moveSeqs instead of movedSeqs.